pages/[league]/schedule.tsx

```diff
@@ -19,11 +26,89 @@ function Schedule({ league }: Props): JSX.Element {
         }}
       />
       <Header league={league} activePage="schedule" />
-      <div>This is a placeholder</div>
+      <Container>
+        <SeasonTypeSelectContainer role="tablist">
+          <SeasonTypeSelectItem
+            key={"Pre-Season"}
+            onClick={() => setSeasonType(() => "Pre-Season")}
+            active={seasonType === "Pre-Season"}
+            tabIndex={0}
+            role="tab"
+            aria-selected={seasonType === "Pre-Season"}
+          >
+            {"Pre-Season"}
```

Comment:
Is there a reason we can't use these normally without the brackets and quotation marks?

Reply:
Nope. I had a variable there before while working on it and didn't think to remove the brackets and quotes when I started using a string. I'll remove them